### PR TITLE
[BUG] Truncated long urls in /account/statistics page

### LINKED ISSUE #8952: [BUG] In the statistics, design breaks with long links
### Description

If you have some long link in your link list, on the panel that shows click number on each link, the link line doesn't break so the card containing the links get larger and larger.

### Screenshots

Screenshot 2023-09-04 164133

### Do you want to work on this issue?

Yes

### Additional information

To fix this it would be enough to wrap the content or to add an overflow handling. Since those aren't clickable link but just informative, I feel like the wrap solution would work better, what do you think?

### PR DESCRIPTION
**Pull request by theunhackable**
## Fixes Issue
Closes #8952

## Changes proposed
- Fixed unresponsiveness and overflow of long links
- Added expanding and collapsing feature

## Change requested
- requested to truncate the link instead of the complex solution.

## Final changes

- added truncate class from tailwindCSS as per the recomendations and solve the overflow issue by adding max-width.
- 

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots
Truncated link 👇:
image

## Note to reviewers

Responsiviness if fixed. I have not seen any issue raised on this so i fixed if without raising an issue. If there are any open issues that I am unable to see about the responsiveness of the link please close it too.

### CODE REVIEW

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Thank you for the contribution but I think we should try to achieve this with css as this looks over complicated

**Comment by theunhackable** [comment]
One thing I could do is just to limit the link to certain fixed characters. Can you please eloborate your way of using CSS in this? Any reference would help

**Comment by eddiejaoude** [comment]
Here are the docs to Tailwind, we can benefit from the tool we use without needing to code it all rom scratch https://tailwindcss.com/docs/text-overflow

File: pages/account/statistics.js [hunk 1]
```diff
@@ -272,7 +272,7 @@ export default function Statistics({ data, profile, progress, BASE_URL }) {
             {data.links &&
               data.links.individual.map((link) => (
                 <tr key={link.url}>
-                  <td className="md:whitespace-nowrap py-4 pl-4 pr-3 text-sm font-medium text-primary-high dark:text-primary-low sm:pl-6">
+                  <td className=" truncate break-words max-sm:max-w-[200px] max-w-lg bg-dark py-4 pl-4 pr-3 text-sm font-medium text-primary-high dark:text-primary-low">
```
Comment (eddiejaoude) [inline pages/account/statistics.js:275]: Is the px width needed? Seems fragile, it should automatically truncate on overflow

Comment (theunhackable) [reply pages/account/statistics.js:275]: it doesn't interfere with any modern devices as 99.99999999% devices are above width 320px. I'm just making sure that there is atleast some breathing area is given to the link inorder to make it readable. I don't think it is a problem.

Comment (eddiejaoude) [reply pages/account/statistics.js:275]: px measurement is not great, if not needed most of the time best to remove it - we don't need to over optimise

Comment (theunhackable) [reply pages/account/statistics.js:275]: image
 
For some strange reason the table is overflowing after removing the breakpoint.

**Comment by eddiejaoude** [comment]
Thank you for your contributing but I will close this as there has been no activity for 4+ weeks and there are conflicts

**Pull request closed without merging**